src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/tests/infrastructure/TokenizedContentProvider.ts

```diff
@@ -4,11 +4,21 @@
  * ------------------------------------------------------------------------------------------ */
 
 import * as fs from 'fs';
+import * as path from 'path';
+import * as oniguruma from 'vscode-oniguruma';
 import { IGrammar, INITIAL, IRawGrammar, ITokenizeLineResult, parseRawGrammar, Registry } from 'vscode-textmate';
 import { ITokenizedContent } from './ITokenizedContent';
 
 let razorGrammarCache: IGrammar | undefined;
 
+const wasmBin = fs.readFileSync(path.join(__dirname, '../../node_modules/vscode-oniguruma/release/onig.wasm')).buffer;
+const vscodeOnigurumaLib = oniguruma.loadWASM(wasmBin).then(() => {
```

Comment:
This is required due to the newer version of the oniguruma API we're now forced to use.